src/linux/system.rs

```diff
@@ -519,7 +519,20 @@ impl SystemExt for System {
         }
     }
 
-    fn get_version(&self) -> Option<String> {
+    fn get_kernel_version(&self) -> Option<String> {
+        let mut s = String::new();
+
+        if File::open("/proc/version")
```

Comment:
Just thought about it but there is actually a function which wraps all the `File::open` and `read_to_string` which is called... [read_to_string](https://doc.rust-lang.org/nightly/std/fs/fn.read_to_string.html) haha

Reply:
added in 81c7eb0